R/add_cols_result_df_pd_changes.R

```diff
@@ -0,0 +1,61 @@
+#' Function that renames Merton model baseline result columns and adds additional
+#' columns for the second application of the model in the late & sudden scenario.
+#'
+#' @param data A dataframe that contains the result data from the baseline
+#'   Merton model
+#' @param horizon A cahracter vector of length 1 indicating whether to return
+#'   a result obejct for overall or yearly results. Must be "annual" or "overall"
+#'
+#' @family helper functions
+#'
+#' @export
+
+add_cols_result_df_pd_changes <- function(data,
+                                          horizon = NULL) {
+
+  force(data)
+  horizon %||% stop("Must provide input for 'horizon'", call. = FALSE)
+
+  horizon_allowed <- horizon %in% c(
+    "annual",
+    "overall"
+  )
+  stopifnot(horizon_allowed)
+
+  if (horizon == "overall") {
```

Comment:
Remember if takes an atomic input but `==` can return a longer vector. You may check the length of `horizon` before `if` or use `identical()` inside `if`.